### ISSUE TITLE
cannot use `import from 'approximate-number'` 

### ISSUE THREAD

**Issue by filgut1** [nfriedly/approximate-number#22]

**Comment by nfriedly** [comment]
Yeah, I think this library completely predates `import` and `export`.

If you want to open a PR that either adds an ESM wrapper or else converts the library over to ESM entirely, I'd appreciate it.

I have a slight preference for the wrapper option due to better backwards compatibility, but we can just call it 3.x if we change everything over.

**Comment by nfriedly** [comment]
Fixed in version 3.0.0
